examples/AvroBlogExamples/Program.cs

```diff
@@ -0,0 +1,148 @@
+﻿// Copyright 2018 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Avro;
+using Avro.Generic;
+using Confluent.Kafka;
+using Confluent.Kafka.Serialization;
+
+
+namespace AvroBlogExample
+{
+    /// <summary>
+    ///     Complete source for the examples programs discussed in the blog post:
+    ///     [insert blog URL here]
+    /// </summary>
+    class Program
+    {
+        static void ProduceGeneric(string bootstrapServers, string schemaRegistryUrl)
+        {
+            var config = new Dictionary<string, object>
+            {
+                { "bootstrap.servers", bootstrapServers },
+                { "schema.registry.url", schemaRegistryUrl }
+            };
+
+            using (var producer = new Producer<Null, GenericRecord>(config, null, new AvroSerializer<GenericRecord>()))
+            {
+                var logLevelSchema = (EnumSchema)Schema.Parse(
+                    File.ReadAllText("LogLevel.asvc"));
+
+                var logMessageSchema = (RecordSchema)Schema
+                    .Parse(File.ReadAllText("LogMessage.V1.asvc")
+                        .Replace(

```

Comment:
this stuff warrants a comment, what is going on here?

Reply:
this is discussed at length in the blog post, don't think it needs to be put here as well.

Reply:
Okay